packages/label-sync/src/label-sync.ts

```diff
@@ -38,18 +38,37 @@ interface Repos {
   ];
 }
 
+interface GetApiLabelsResponse {
+  apis: Array<{
+    display_name: string; // Access Approval
+    github_label: string; // api: accessapproval
+    api_shortname: string; // accessapproval
+  }>;
+}
+
+interface PublicReposResponse {
+  repos: Array<{
+    repo: string;
+    github_label: string;
+  }>;
+}
+
+interface HandlerFunction extends ApplicationFunction {
+  getApiLabels: Function;
+}
+
 // Labels are fetched by reaching out to GitHub *instead* of grabbing the file
 // from the local copy.  We are using the `PushEvent` to detect the change,
 // meaning the file running in cloud will be older than the one on master.
 let labelsCache: Labels;
 async function getLabels(github: GitHubAPI, repoPath: string): Promise<Labels> {
   if (!labelsCache) {
-    await refreshLabels(github, repoPath);
+    await refreshLabels(github);
   }
   const labels = {
     labels: labelsCache.labels.slice(0),
   } as Labels;
-  const apiLabelsRes = await handler.getApiLabels(repoPath);
```

Comment:
this method is on `handler` so that it can easily be mocked in tests, setting up 75 nock calls is a pain in the butt for simple methods like this.